src/User32.Desktop/User32.cs

```diff
@@ -144,6 +144,17 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern unsafe IntPtr SendMessage(IntPtr hWnd, int wMsg, void* wParam, void* lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern unsafe IntPtr SendMessage(IntPtr hWnd, WindowMessage wMsg, void* wParam, void* lParam);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern unsafe bool PostMessage(IntPtr hWnd, int Msg, void* wParam, void* lParam);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern unsafe bool PostMessage(IntPtr hWnd, WindowMessage wMsg, void* wParam, void* lParam);
```

Comment:
Same here: can you move the `int Msg` overload to the Helper file and make it call this other method to keep just 1 P/Invoke per function?
Allso, on the `WindowMessage wMsg` overload, where did the `w` prefix come from? I don't see that in the MSDN doc.